components/cam/bld/build-namelist

```diff
@@ -4016,7 +4025,7 @@ if ($cfg->get('dyn') =~ /se/) {
                   statefreq se_partmethod se_topology se_ftype
                   integration nu nu_div nu_p nu_q nu_top  se_phys_tscale
                   vert_remap_q_alg se_limiter_option qsplit rsplit tstep_type
-                  hypervis_scaling mesh_file vthreads);
+                  hypervis_scaling mesh_file vthreads cubed_sphere_map);
```

Comment:
Does this bring map parameter into cam namelist? By default right now there is no map in the list and homme defines which map to use internally. It is preferred to not to introduce extra params into namelist. Is there a way to add map to cam nl based on resolution?

Reply:
Per our phone conversation, I removed this and added a section to the "namelist_mod.F90" in the HOMME code that handles the requirement that cubed_sphere_map=2 for fv_nphys>0.